test/unit/gateways/authorize_net_test.rb

```diff
@@ -523,6 +546,7 @@ def test_truncation
       order_id: "a" * 21,
       description: "a" * 256,
       billing_address: address(
+        name: ["a" * 51, "a" * 51].join(" "),
```

Comment:
I had to make this change in order to give the addresses being used in these tests actual values; by default they're set to "Jim Smith" but were falling through to the billing address of the credit card.